packages/editor/src/components/properties/RenderSettingsEditor.tsx

```diff
@@ -107,19 +110,37 @@ const ShadowTypeOptions = [
 
 export const RenderSettingsEditor: EditorComponentType = (props) => {
   const { t } = useTranslation()
-  const rendererState = useComponent(props.entity, RenderSettingsComponent)
+  const rendererSettingsState = useComponent(props.entity, RenderSettingsComponent)
+  const rendererState = useHookstate(getMutableState(RendererState))
 
   return (
     <PropertyGroup
       name={t('editor:properties.renderSettings.name')}
       description={t('editor:properties.renderSettings.description')}
     >
+      <InputGroup
+        name="Use Post Processing"
```

Comment:
Not sure if its a convention or so. But I think spaces & uppercaps are not recommended in `name` and `id` attributes:

https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html

Reply:
Yeah I agree and I followed the code to maintain uniformity